server/src/github.ts

```diff
@@ -108,6 +114,28 @@ export class GitHub {
       yield result.data;
     }
   }
+
+  // tslint:disable-next-line:no-any
+  async get(path: string, userToken: string, parseJSON = true): Promise<any> {
```

Comment:
I think you could return `Promise<object|string>`.

Reply:
It's not a string, its a response object from the request library, so I'm not sure how to type it. `request.get()` returns an any.